sdk/python/foundation-models/system/inference/text-to-image/aacs-prepare-uai.ipynb

```diff
@@ -0,0 +1,434 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Prepare UAI account for Azure AI Content Safety (AACS) enabled text-to-image model deployment\n",
```

Comment:
What does UAI stand for ? Can we have full form mentioned just once. Rest everywhere we can use acronym

Reply:
Thanks, Added it.